### PR TITLE
nft family sorting tweaks

### PR DESCRIPTION
**Pull request by skylarbarrera**
PoW: Screen Shot 2021-09-27 at 10 20 49 AM

### CODE REVIEW

**Review by BirinIhor — COMMENTED** [review]
e7e682ee748dc045e9088319c511e4f71ddd111b

File: src/helpers/assets.js [hunk 1]
```diff
@@ -230,7 +230,7 @@ export const buildUniqueTokenList = (uniqueTokens, selectedShowcaseTokens) => {
     });
   }
 
-  rows = sortBy(rows, ['familyName']);
+  rows = sortBy(rows, row => row.familyName.replace('The ', '').toLowerCase());
```
Comment (nicoglennon) [inline src/helpers/assets.js:233]: Could use a more general regex to catch all the "the"s: 
```
const theRegex = /^(the|The|THE)\s/g;
rows = sortBy(rows, row => row.familyName.replace(theRegex, '').toLowerCase());
```

**Review by nicoglennon — APPROVED** [review]
🏎

File: src/helpers/assets.js (hunk 1 above)
Comment (skylarbarrera) [reply src/helpers/assets.js:233]: ty ser 🤝

**Pull request merged**